File: Functions/5.cpp
#include<bits/stdc++.h>
using namespace std;

void check_prime(int n){
    if(n==1){
        cout << "Neither Prime nor Composite";
        return;
    }
    for(int i=2;i<sqrt(n)+1;i++){
        if(n%i==0){
            cout << "A Composite Number";
            return;
        }
    }

    cout << "A Prime Number";
    return;
}

int main(){
    int n;
    cout<<"Enter a Number: ";
    cin>>n;
    check_prime(n);
}

File: Functions/5.py
def prime_or_not(n):
    if(n==1):
        print("Neither Prime nor Composite")
        return
    for i in range(2,int(n**0.5) + 1):
        if(n%i == 0):
            print("A Composite Number")
            break
    else:
        print("Prime Number")

n = int(input("Enter a Number: "))
prime_or_not(n)